### PR TITLE
move all EOL comments on typepath names to dmdoc

### PR DESCRIPTION
**Pull request by warriorstar-orion**
## What Does This PR Do
This PR moves all EOL comments on typepath declarations to DMDoc comments on the preceding line.
For reproduction, the regexes were `(^/[^/].*/\w+)(\s+)//\s*(.*)$` -> `/// $3\n$1`
## Why It's Good For The Game
A regex search for `^/foo$` is the fastest way to find a declaration, but EOL comments cause this to fail.
## Testing
Visual inspection.
## Changelog
NPFC

### CODE REVIEW

File: code/modules/mob/new_player/sprite_accessories/unathi/unathi_hair.dm [hunk 1]
```diff
@@ -3,7 +3,8 @@
 	species_allowed = list("Unathi")
 	glasses_over = 1
 
-/datum/sprite_accessory/hair/braid_hip_una // Future coder: Change the typepath under unathi. This is a legacy of my accessories splitting PR where typepath change is avoided at all cost
+/// Future coder: Change the typepath under unathi. This is a legacy of my accessories splitting PR where typepath change is avoided at all cost
```
Comment (lewcc) [inline code/modules/mob/new_player/sprite_accessories/unathi/unathi_hair.dm:6]:
```suggestion
// Future coder: Change the typepath under unathi. This is a legacy of my accessories splitting PR where typepath change is avoided at all cost
```

File: code/modules/projectiles/ammunition/magazines.dm [hunk 2]
```diff
@@ -589,7 +590,8 @@
 /obj/item/ammo_box/magazine/toy/smgm45/riot
 	ammo_type = /obj/item/ammo_casing/caseless/foam_dart/riot
 
-/obj/item/ammo_box/magazine/detective/speedcharger //yes this doesn't really belong here but nowhere else works
+/// yes this doesn't really belong here but nowhere else works
```
Comment (lewcc) [inline code/modules/projectiles/ammunition/magazines.dm:593]:
```suggestion
// yes this doesn't really belong here but nowhere else works
```

File: code/modules/reagents/reagent_containers/spray.dm [hunk 3]
```diff
@@ -274,7 +274,8 @@
 
 
 // Plant-B-Gone
```
Comment (lewcc) [inline code/modules/reagents/reagent_containers/spray.dm:276]:
```suggestion
/// Plant-B-Gone
```

File: code/modules/surgery/organs/eyes.dm [hunk 4]
```diff
@@ -211,7 +211,8 @@
 #define ONE_SHATTERED 1
 #define BOTH_SHATTERED 2
 
-/obj/item/organ/internal/eyes/cybernetic/eyesofgod //no occuline allowed
+/// no occuline allowed
```
Comment (lewcc) [inline code/modules/surgery/organs/eyes.dm:214]:
```suggestion
// no occuline allowed
```

File: code/modules/surgery/organs/augments_internal.dm [hunk 5]
```diff
@@ -240,7 +240,8 @@
 	REMOVE_TRAIT(M, TRAIT_COMIC_SANS, "augment")
 	return ..()
 
-/obj/item/organ/internal/cyberimp/brain/speech_translator //actual translating done in human/handle_speech_problems
+/// actual translating done in human/handle_speech_problems
```
Comment (lewcc) [inline code/modules/surgery/organs/augments_internal.dm:243]:
```suggestion
// actual translating done in human/handle_speech_problems
```

**Review by lewcc — COMMENTED** [review]
Some quick things that don't seem like they should be autodocs

**Review by lewcc — APPROVED** [review]
Looks good to me, would be nice to get another set of eyes on it just in case

File: code/modules/surgery/organs/augments_arms.dm [hunk 6]
```diff
@@ -337,7 +338,8 @@
 	action_icon_state = list(/datum/action/item_action/organ_action/toggle = "janibelt")
 	emp_proof = TRUE
 
-/obj/item/organ/internal/cyberimp/arm/janitorial/advanced/l /// its for ERT, but still probably a good idea.
+/// / its for ERT, but still probably a good idea.
```
Comment (S34NW) [inline code/modules/surgery/organs/augments_arms.dm:341]:
```suggestion
/// its for ERT, but still probably a good idea.
```

File: code/modules/surgery/organs/augments_arms.dm [hunk 7]
```diff
@@ -328,7 +328,8 @@
 	parent_organ = "l_arm"
 	slot = "l_arm_device"
 
-/obj/item/organ/internal/cyberimp/arm/janitorial/advanced /// ERT implant, i dont overly expect this to get into the hands of crew
+/// / ERT implant, i dont overly expect this to get into the hands of crew
```
Comment (S34NW) [inline code/modules/surgery/organs/augments_arms.dm:331]:
```suggestion
/// ERT implant, i dont overly expect this to get into the hands of crew
```

File: code/modules/surgery/organs/augments_internal.dm [hunk 8]
```diff
@@ -240,7 +240,8 @@
 	REMOVE_TRAIT(M, TRAIT_COMIC_SANS, "augment")
 	return ..()
 
-/obj/item/organ/internal/cyberimp/brain/speech_translator //actual translating done in human/handle_speech_problems
+// actual translating done in human/handle_speech_problems
```
Comment (S34NW) [inline code/modules/surgery/organs/augments_internal.dm:243]:
```suggestion
/// actual translating done in human/handle_speech_problems
```

File: code/modules/surgery/organs/eyes.dm [hunk 9]
```diff
@@ -211,7 +211,8 @@
 #define ONE_SHATTERED 1
 #define BOTH_SHATTERED 2
 
-/obj/item/organ/internal/eyes/cybernetic/eyesofgod //no occuline allowed
+// no occuline allowed
```
Comment (S34NW) [inline code/modules/surgery/organs/eyes.dm:214]:
```suggestion
/// no occuline allowed
```

**Review by S34NW — COMMENTED** [review]
one or two other oddities, then LGTM

**Pull request merged**